pkg/controller/volume/persistentvolume/framework_test.go

```diff
@@ -756,6 +759,7 @@ func newClaim(name, claimUID, capacity, boundToVolume string, phase v1.Persisten
 			},
 			VolumeName:       boundToVolume,
 			StorageClassName: class,
+			VolumeMode:       &fs,
```

Comment:
why was this change needed?

Reply:
still outstanding

Reply:
still required, i added a revert commit then dropped it again after @msau42 explained